README.md

```diff
@@ -1,5 +1,9 @@
 # IPFS Specs
 
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freejs-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
```

Comment:
It is actually `freenode` and not `freejs`, that was a outcome of the great rename of 2015 (node-> js)